### ISSUE TITLE
protractor 'advertises' its --verbose option, but cannot use that

### ISSUE THREAD

**Issue by attila123** [angular/protractor#4196]
- Node Version: `v6.9.5`
- Protractor Version: `5.1.1`
- Operating System and Version `Linux Mint 18.1`
- Steps to reproduce the bug

`protractor --help` tells about the --verbose option:
```
  --verbose                              Print full spec names
```
I wanted to try how it works. So I executed protractor with this flag, but apparently it cannot make sense of this `--verbose` option, see the warning below:

```
$ protractor protractor-conf.js --verbose
[14:52:16] I/launcher - Running 1 instances of WebDriver
[14:52:16] I/hosted - Using the selenium server at http://localhost:4444/wd/hub
[14:52:17] W/runner - Ignoring unknown extra flags: verbose. This will be an error in future versions, please use --disableChecks flag to disable the  Protractor CLI flag checks. 
Protractor animation is disabled
Protractor synchronization is disabled
(...)
```
So maybe this `--verbose` is a half-way done feature? I guess it should be completed or removed.

**Comment by NickTomlin** [comment]
I can reproduce this as well; I pr'd the fix in #4200

**Comment by attila123** [comment]
Thanks @NickTomlin ! :)
